src/main/java/org/broadinstitute/hellbender/utils/pairhmm/VectorLoglessPairPDHMM.java

```diff
@@ -0,0 +1,399 @@
+        nativeArguments.avxLevel = AVXLevel.FASTEST_AVAILABLE;
+        nativeArguments.setMaxMemoryInMB(1024);
+        nativeArguments.openMPSetting = OpenMPSetting.FASTEST_AVAILABLE;
+        pairPDHmm.initialize(nativeArguments);
+        // TODO no initialize argument?
+        // pairPDHmm.initialize(args);
+    }
+
+    public void initialize(final int readMaxLength, final int haplotypeMaxLength) {
+        super.initialize(readMaxLength, haplotypeMaxLength);
+    }
+    // /**
+    // * {@inheritDoc}
+    // */
+    //// @Override
+    // public void initialize(final int readMaxLength, final int haplotypeMaxLength
+    // ) {
+    // super.initialize(readMaxLength, haplotypeMaxLength);
+    //
+    // branchMatchMatrix = new
+    // double[paddedMaxReadLength][paddedMaxHaplotypeLength];
+    // branchInsertionMatrix = new
+    // double[paddedMaxReadLength][paddedMaxHaplotypeLength];
+    // branchDeletionMatrix = new
+    // double[paddedMaxReadLength][paddedMaxHaplotypeLength];
+    // // do not need to call super.initialize()
+    // int numHaplotypes = haplotypes.size();
+    // mHaplotypeDataArray = new HaplotypeDataHolder[numHaplotypes];
+    // int idx = 0;
+    // haplotypeToHaplotypeListIdxMap.clear();
+    // for (final Haplotype currHaplotype : haplotypes) {
+    // mHaplotypeDataArray[idx] = new HaplotypeDataHolder();
+    // mHaplotypeDataArray[idx].haplotypeBases = currHaplotype.getBases();
+    // haplotypeToHaplotypeListIdxMap.put(currHaplotype, idx);
+    // ++idx;
+    // }
+    // }
+
+    // /**
+    // * {@inheritDoc}
+    // */
+    // @Override
+    // public void initialize(final List<Haplotype> haplotypes, final Map<String,
+    // List<GATKRead>> perSampleReadList,
+    // final int readMaxLength, final int haplotypeMaxLength) {
+    //
+    // }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void computeLog10Likelihoods(final LikelihoodMatrix<GATKRead, PartiallyDeterminedHaplotype> logLikelihoods,
+                                        final List<GATKRead> processedReads,
+                                        final PairHMMInputScoreImputator inputScoreImputator,
+                                        final int rangeForReadOverlapToDeterminedBases) {
+
+        if (processedReads.isEmpty()) {
+            return;
+        }
+        if (doProfiling) {
+            startTime = System.nanoTime();
+        }
+        // (re)initialize the pairHMM only if necessary
+        final int readMaxLength = findMaxReadLength(processedReads);
+        final int haplotypeMaxLength = findMaxAlleleLength(logLikelihoods.alleles());
+        final int readCount = processedReads.size();
+        final List<PartiallyDeterminedHaplotype> alleles = logLikelihoods.alleles();
+        final int alleleCount = alleles.size();
+        final int totalComps = readCount * alleleCount;
+        // logger.info("Total Reads: " + readCount + "Total Alleles: " + alleleCount +
```

Comment:
Is this something that should be kept in and logged at trace or debug level? or should we drop it?

Reply:
I added a line for "total comparisons" to the end of the on close info squak from this class